pycheribuild/projects/cross/openssl.py

```diff
@@ -51,16 +50,23 @@ def setup(self):
         self.configure_args.append(str(self.source_dir / "Configure"))
         self.configure_args.append("BSD-generic64")
         self.configure_args.append("-shared")
-        self.configure_args.append("--install-prefix=" + str(self.destdir))
+	# XXX: should be based on version
+        if (self.project_name != "fett-openssl"):
+            self.configure_args.append("--prefix=" + str(self.destdir) + str(self._install_prefix))
```

Comment:
No parens, and looking at `self.project_name` is ugly

Reply:
I'm hoping to purge the FETT bits in October...